### PR TITLE
feat: Show "create a group" to first-time users (WEBAPP-5266)

### PR DESCRIPTION
**Pull request by bennyn**
When being a new user on Wire, our app asks you to "Start a conversation or create a group" but there is no button to create a group if you have no contacts on Wire. 

**Status quo:**

a

Since we can create a group with ourselves, it makes sense to show a "Create a group" option so that first time users can see our feature set and start a playground conversation.

### CODE REVIEW

File: app/page/template/list/start-ui.htm [hunk 1]
```diff
@@ -114,6 +114,12 @@
   </div>
 
   <!-- ko if: showNoContacts() -->
+    <ul class="start-ui-create-group left-list-items">
+      <li class="left-list-item left-list-item-clickable" data-bind="click: clickOnCreateGroup" data-uie-name="go-create-group">
+        <group-icon class="left-column-icon"></group-icon>
+        <div class="center-column" data-bind="l10n_text: z.string.searchCreateGroup"></div>
+      </li>
+    </ul>
```
Comment (gregor) [inline app/page/template/list/start-ui.htm]: Can we find a solution that does not require duplication of the html?

Comment (bennycode) [reply app/page/template/list/start-ui.htm]: The Andymator and I found a CSS solution to the problem. 👍

File: app/style/list/start-ui.less [hunk 2]
```diff
@@ -63,13 +63,20 @@
   overflow: hidden;
   flex: 1 1;
   flex-direction: column;
+  &.split-view {
```
Comment (gregor) [inline app/style/list/start-ui.less]: We usually have an empty line above new declarations

Comment (bennycode) [reply app/style/list/start-ui.less]: 👍

**Pull request merged**